### PR TITLE
chore: update react related dependencies

### PR DESCRIPTION
**Pull request by nataliepina**
Separated out these react-related dependencies, which required some snapshot updates. Components are still working as expected and all tests are passing. 

## Checklist for PR creator

- [ ] If any new components were added, there are exported from `packages/index.ts`
- [ ] If this PR is associated with a JIRA, it is mentioned in commit message footer ("Closes …")
- [ ] If this PR contains breaking changes, is stated in commit message body ("BREAKING CHANGE: …")
- [x] Info for applicable sections above is provided

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,12 +54,12 @@
     "react-chartist": "0.14.4",
     "react-click-outside": "3.0.1",
     "react-delegate-component": "1.0.0",
-    "react-draggable": "3.3.2",
```
Comment (mstruebing) [inline package.json:57]: Are we even using this dependency?

Comment (nataliepina) [reply package.json:57]: I was unsure about this one, didn't see it actively used here but I saw it in Kommander and wasn't sure if it's safe to remove. Thoughts?

Comment (mstruebing) [reply package.json:57]: In kommander-ui I can see it only in `package-lock.json` and a markdown file. So I think it is not really used and should be safe to be removed.

Comment (nataliepina) [reply package.json:57]: Done! Thanks @mstruebing

**Review by mstruebing — APPROVED** [review]
🎉 Good stuff

**Review by seialkali — APPROVED** [review]
Looks good 🕺

**Pull request merged**